K2Field.K2NE.ServiceBroker/Helpers/PowerShell/PowerShellSerializationHelper.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Management.Automation;
+using System.Runtime.Serialization;
+using System.IO;
+using System.Runtime.Serialization.Json;
+
+
+namespace K2Field.K2NE.ServiceBroker.Helpers.PowerShell
+{
+    public static class PowerShellSerializationHelper
```

Comment:
I'd suggest that every method in this class needs some exception handling. It would be good if we catch it here so we can clearly identify what is failing. What happens if we input wrong json?